pkgs/build-support/cc-wrapper/gnat-wrapper.sh

```diff
@@ -18,7 +24,6 @@ source @out@/nix-support/utils.sh
 # Figure out if linker flags should be passed.  GCC prints annoying
 # warnings when they are not needed.
 dontLink=0
```

Comment:
This is a dead variable, but used in commented code, so I guess I'll leave it for now.